source/extensions/transport_sockets/tls/context_impl.cc

```diff
@@ -911,6 +911,12 @@ ServerContextImpl::ServerContextImpl(Stats::Scope& scope,
   if (config.tlsCertificates().empty()) {
     throw EnvoyException("Server TlsCertificates must have a certificate specified");
   }
+
+  // Compute the session context ID hash. We use all the certificate identities,
+  // since we should have a common ID for session resumption no matter what cert
+  // is used. We do this early because it can throw an EnvoyException.
+  const SessionContextID session_id = generateHashForSessionContexId(server_names);
```

Comment:
As pointed in previous PR (after merge), do you mind fix this spell?

Reply:
Yikes, I stared at this for 5 mins before I could see the problem :)

Reply:
Fixed.